### ISSUE TITLE
About training autoencoder and feadepth

### ISSUE THREAD

**Issue by dsshim0125** [sconlyshootery/FeatDepth#53]
Hi,

I'm really excited to read your research and code.

I have a question that the encoder and the decoder of the autoencoder is freezer during training depth network or 
is it also trained along with the depth network?

Thank you.

***************************
Additionally, can you give some configuration of training autoencoder such as learning rate (scheduling), augmentation, and the number of epochs?

What normalization should be applied in the raw image to replicate your result such as transforms.Normalize((0.5,0.5,0.5), (0.5,0.5,0.5))

**Comment by sconlyshootery** [comment]
These two manners are all feasible, we provide config files to conduct these two kinds of training, which are config/cfg_kitti_fm.py and config/cfg_kitti_fm_joint.py.
Training settings can be found in config files. Normalization is conducted in https://github.com/sconlyshootery/FeatDepth/blob/64128b03873b27ffa5e99a5cb1712dd8aa15cb0d/mono/model/mono_fm/depth_encoder.py#L37

**Comment by dsshim0125** [comment]
Thank you for your support!
